contracts/Auctioneer.sol

```diff
@@ -146,4 +149,11 @@ contract Auctioneer is CloneFactory {
     ///         already closed and funds from the coverage pool are seized.
     /// @dev Override this function to act on auction closed as fully filled.
     function onAuctionFullyFilled(Auction auction) internal virtual {}
+
+    /// @notice Checks the state of the deposit asocciated with an auction
```

Comment:
Can we please change formatting? It seems that all other comments are aligned a bit differently, ie. new lines start directly beneath the first sentence, not under the <@tag> .

Reply:
Sure, Fixed in d5e8369af46610a4d5129dde21d8d0fa85fa7998.